Sources/TokamakCore/MountedViews/MountedApp.swift

```diff
@@ -30,7 +30,7 @@ final class MountedApp<R: Renderer>: MountedCompositeElement<R> {
     // They also have no parents, so the `parent` argument is discarded as well.
     let childBody = reconciler.render(mountedApp: self)
 
-    let child: MountedElement<R> = mountChild(childBody)
+    let child: MountedElement<R> = mountChild(reconciler.renderer, childBody)
```

Comment:
`renderer` instances now need to be explicitly pass through in a lot more places. This allows mounted elements to call back to the renderer to determine whether a given view is primitive for this renderer.